### PR TITLE
Open haptics

### PR DESCRIPTION
**Pull request by sjh26**
Cmake support for Open Haptics Libraries

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -557,6 +558,13 @@ IF(PLUS_USE_THORLABS_VIDEO)
   ENDIF()
 ENDIF()
 
+IF(PLUS_USE_OPENHAPTICS)
+  FIND_PACKAGE(OpenHaptics)
+  IF(NOT OPENHAPTICS_FOUND)
```
Comment (adamrankin) [inline CMakeLists.txt:563]: Generally, with FindPackage modules, the case of the resulting variables should match the case of the Find argument.

In this case, `FIND_PACKAGE(OpenHaptics)` should produce variables of the form `OpenHaptics_`
or
`FIND_PACKAGE(OPENHAPTICS) --> OPENHAPTICS_`

File: Modules/FindOpenHaptics.cmake [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+# Copyright Iowa State University 2009-2010.
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 2.6.3)
+
+
+SET( OpenHaptics_PATH_HINTS
+  "C:/OpenHaptics/Developer/3.4.0"
```
Comment (adamrankin) [inline Modules/FindOpenHaptics.cmake:59]: Could you also add common linux paths

File: Modules/FindOpenHaptics.cmake [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+  )
+
+
+FIND_PATH( OpenHaptics_ROOT_DIR Geomagic_OpenHaptics.ico
```
Comment (adamrankin) [inline Modules/FindOpenHaptics.cmake:63]: Does this file also exist on linux?

Comment (sjh26) [reply Modules/FindOpenHaptics.cmake:63]: I will double-check that, and add linux paths

**Comment by adamrankin** [comment]
I've cleaned up the find module and some of the CMake stuff, it's finding everything properly and seems correct. Testing build now.

File: Modules/FindOpenHaptics.cmake (hunk 3 above)
Comment (adamrankin) [reply Modules/FindOpenHaptics.cmake:63]: Let me know, and I'll add the fixes (Just paste them as a reply to this), as I've made changes in the master

Comment (sjh26) [reply Modules/FindOpenHaptics.cmake:63]: So, the linux sdk is structured very differently from the windows and so I will need to rewrite a bunch to support linux compiling.  Will get back to you when that is done.

Comment (adamrankin) [reply Modules/FindOpenHaptics.cmake:63]: Well, honestly it's up to you. Linux support would be nice, but not necessary.

**Pull request merged**